### PR TITLE
Add UI to display contacts

### LINKED ISSUE #7: Create list contacts UI

### PR DESCRIPTION
**Pull request by sahilLikes2Code**
Closes #7

### CODE REVIEW

**Comment by sahilLikes2Code** [comment]
@karthiknmenon Please review.

File: app/javascript/src/components/Dashboard/Contacts/ContactsTable.jsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import React from "react";
+
+import { MenuHorizontal } from "@bigbinary/neeto-icons";
+import { Typography } from "@bigbinary/neetoui/v2";
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/ContactsTable.jsx:4]: Please use aliases.

File: app/javascript/src/components/Dashboard/Contacts/ContactsTable.jsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+import "dayjs";
+import { Checkbox } from "neetoui";
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/ContactsTable.jsx:6]: Isn't this supposed to be `/v2`.

File: app/javascript/src/components/Dashboard/Contacts/ContactsTable.jsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+import dummyContacts from "../../../constants/dummyContacts";
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/ContactsTable.jsx:8]: Keep contacts constants in the scope of this directory.

File: app/javascript/src/components/Dashboard/Contacts/ContactsTable.jsx [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+              </td>
+              <td>{contact.email}</td>
+              <td>Feb 5, 2021</td>
+              <td>
+                <MenuHorizontal />
+              </td>
+            </tr>
+          ))}
+        </tbody>
+      </table>
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/ContactsTable.jsx:66]: Please use `Table` component from neetoUI.

File: app/javascript/src/components/Dashboard/Contacts/Main.jsx [hunk 5]
```diff
@@ -0,0 +1,76 @@
+import React, { useState, useEffect } from "react";
+
+import { Search } from "@bigbinary/neeto-icons";
+import { Button, Input, PageLoader } from "@bigbinary/neetoui/v2";
+import { Header } from "@bigbinary/neetoui/v2/layouts";
+import { Pagination } from "@bigbinary/neetoui/v2";
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/Main.jsx:6]: Please use aliases.

File: app/javascript/src/components/Dashboard/Contacts/Index.jsx [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+const Contacts = () => {
+  return (
+    <>
+      <UnequalTwoColumnSplitter menu={<SideMenu />} mainContent={<Main />} />
+    </>
+  );
+};
+
+export default Contacts;
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/Index.jsx:16]: Why do we need this?

File: app/javascript/src/components/Dashboard/Contacts/Main.jsx [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+
+import notesApi from "apis/notes";
+import ContactsTable from "./ContactsTable";
+
+const Main = () => {
+  const [loading, setLoading] = useState(true);
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/Main.jsx:12]: `const [isLoading, setIsLoading] = useState(true);`

File: app/javascript/src/constants/dummyContacts.js [hunk 8]
```diff
@@ -0,0 +1,54 @@
+const dummyContacts = [
```
Comment (karthiknmenon) [inline app/javascript/src/constants/dummyContacts.js:1]: `export const DUMMY_NOTES`. 
Please move this to the scope of `Notes` folder.

File: app/javascript/src/components/Dashboard/Contacts/SideMenu.jsx [hunk 9]
```diff
@@ -0,0 +1,53 @@
+import React from "react";
+
+import { Search, Settings, Plus } from "@bigbinary/neeto-icons";
+import { Typography } from "@bigbinary/neetoui/v2";
+import { MenuBar } from "@bigbinary/neetoui/v2/layouts";
```
Comment (karthiknmenon) [inline app/javascript/src/components/Dashboard/Contacts/SideMenu.jsx:5]: Please use aliases.

**Review by karthiknmenon — COMMENTED** [review]
@sahilLikes2Code Please check the suggestions.

File: app/javascript/src/components/Dashboard/Contacts/ContactsTable.jsx (hunk 3 above)
Comment (sahilLikes2Code) [reply app/javascript/src/components/Dashboard/Contacts/ContactsTable.jsx:8]: Yes. However, this is no longer required and was removed, as I switched to `Table` from "neetoui/v2" to display the same content.

File: app/javascript/src/components/Dashboard/Contacts/ContactsTable.jsx (hunk 4 above)
Comment (sahilLikes2Code) [reply app/javascript/src/components/Dashboard/Contacts/ContactsTable.jsx:66]: Done sir, in a newer commit.

File: app/javascript/src/components/Dashboard/Contacts/ContactsTable.jsx (hunk 2 above)
Comment (sahilLikes2Code) [reply app/javascript/src/components/Dashboard/Contacts/ContactsTable.jsx:6]: Yes. However, this is no longer required and was removed, as I switched to Table from "neetoui/v2" to display the same content.

File: app/javascript/src/components/Dashboard/Contacts/Index.jsx (hunk 6 above)
Comment (sahilLikes2Code) [reply app/javascript/src/components/Dashboard/Contacts/Index.jsx:16]: Same as [this](https://github.com/sahilLikes2Code/wheel/pull/2#discussion_r755348512). To make it look more clean.

**Pull request merged**